### PR TITLE
[KYUUBI #6251] Improve kyuubi-beeline help message

### LINKED ISSUE #6251: [TASK][EASY] Improve `kyuubi-beeline` help message
### What's the level of this task?

EASY

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before creating

- [X] I have searched in the [task list](https://github.com/orgs/apache/projects/343) and found no similar tasks.

### Mentor

- [X] I have sufficient expertise on this task, and I volunteer to be a mentor of this task to guide contributors through the task.

### Skill requirements

- Basic knowledge of Java and Kyuubi

### Background and Goals

The current `kyuubi-beeline` output message has some errors and is not very pretty.

```
$ bin/kyuubi-beeline --help
Usage: java org.apache.hive.cli.beeline.BeeLine 
   -u <database url>               the JDBC URL to connect to
   -c <named url>                  the named JDBC URL to connect to,
                                   which should be present in beeline-site.xml
                                   as the value of beeline.hs2.jdbc.url.<namedUrl>
   -r                              reconnect to last saved connect url (in conjunction with !save)
   -n <username>                   the username to connect as
   -p <password>                   the password to connect as
   -d <driver class>               the driver class to use
   -i <init file>                  script file for initialization
   -e <query>                      query that should be executed
   -f <exec file>                  script file that should be executed
   -w (or) --password-file <password file>  the password file to read password from
   --hiveconf property=value       Use value for given property
   --hivevar name=value            hive variable name and value
                                   This is Hive specific settings in which variables
                                   can be set at session level and referenced in Hive
                                   commands or queries.
   --property-file=<property-file> the file to read connection properties (url, driver, user, password) from
   --color=[true/false]            control whether color is used for display
   --showHeader=[true/false]       show column names in query results
   --escapeCRLF=[true/false]       show carriage return and line feeds in query results as escaped \r and \n 
   --headerInterval=ROWS;          the interval between which heades are displayed
   --fastConnect=[true/false]      skip building table/column list for tab-completion
   --autoCommit=[true/false]       enable/disable automatic transaction commit
   --verbose=[true/false]          show verbose error messages and debug info
   --showWarnings=[true/false]     display connection warnings
   --showDbInPrompt=[true/false]   display the current database name in the prompt
   --showNestedErrs=[true/false]   display nested errors
   --numberFormat=[pattern]        format numbers using DecimalFormat pattern
   --force=[true/false]            continue running script even after errors
   --maxWidth=MAXWIDTH             the maximum width of the terminal
   --maxColumnWidth=MAXCOLWIDTH    the maximum width to use when displaying columns
   --silent=[true/false]           be more silent
   --autosave=[true/false]         automatically save preferences
   --outputformat=[table/vertical/csv2/tsv2/dsv/csv/tsv]  format mode for result display
                                   Note that csv, and tsv are deprecated - use csv2, tsv2 instead
   --incremental=[true/false]      Defaults to false. When set to false, the entire result set
                                   is fetched and buffered before being displayed, yielding optimal
                                   display column sizing. When set to true, result rows are displayed
                                   immediately as they are fetched, yielding lower latency and
                                   memory usage at the price of extra display column padding.
                                   Setting --incremental=true is recommended if you encounter an OutOfMemory
                                   on the client side (due to the fetched result set size being large).
                                   Only applicable if --outputformat=table.
   --incrementalBufferRows=NUMROWS the number of rows to buffer when printing rows on stdout,
                                   defaults to 1000; only applicable if --incremental=true
                                   and --outputformat=table
   --truncateTable=[true/false]    truncate table column when it exceeds length
   --delimiterForDSV=DELIMITER     specify the delimiter for delimiter-separated values output format (default: |)
   --isolation=LEVEL               set the transaction isolation level
   --nullemptystring=[true/false]  set to true to get historic behavior of printing null as empty string
   --maxHistoryRows=MAXHISTORYROWS The maximum number of rows to store beeline history.
   --delimiter=DELIMITER           set the query delimiter; multi-char delimiters are allowed, but quotation
                                   marks, slashes, and -- are not allowed; defaults to ;
   --convertBinaryArrayToString=[true/false]    display binary column data as string or as byte array 
   --help                          display this message
 
   Example:
    1. Connect using simple authentication to HiveServer2 on localhost:10000
    $ beeline -u jdbc:hive2://localhost:10000 username password

    2. Connect using simple authentication to HiveServer2 on hs.local:10000 using -n for username and -p for password
    $ beeline -n username -p password -u jdbc:hive2://hs2.local:10012

    3. Connect using Kerberos authentication with hive/localhost@mydomain.com as HiveServer2 principal
    $ beeline -u "jdbc:hive2://hs2.local:10013/default;principal=hive/localhost@mydomain.com"

    4. Connect using SSL connection to HiveServer2 on localhost at 10000
    $ beeline "jdbc:hive2://localhost:10000/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"

    5. Connect using LDAP authentication
    $ beeline -u jdbc:hive2://hs2.local:10013/default <ldap-username> <ldap-password>
 
Usage: java org.apache.hive.beeline.KyuubiBeeLine
   --python-mode                   Execute python code/script.
```

### Implementation steps

- `kyuubi-beeline` is derived from Hive `beeline`, we have made some changes since forking, some reference to Hive is improper now and we should remove them
- replace `jdbc:hive2//` with `jdbc:kyuubi//`
- update and validate examples
- other changes

### Additional context

Introduction of [2024H1 Kyuubi Code Contribution Program](https://github.com/apache/kyuubi/issues/6232)

### PR DESCRIPTION
**Pull request by dupen01**
# :mag: Description
## Issue References 🔗

This pull request fixes #6251

## Describe Your Solution 🔧
As `kyuubi-beeline` is derived from Hive `beeline`, I haven't made extensive modifications to its help message, only adjusting some formatting to enhance its appearance. Below are the specific changes:
1. Replace `jdbc:hive2//` with `jdbc:kyuubi//`.
2. Replace `beeline` with `kyuubi-beeline`.
3. Capitalize the first letter.
4. If the comment for a parameter spans multiple lines, add an additional `\n` at the end of the comment to separate it from the following line.
5. Append the help information for `--python-mode` to the front of `--help`.
6. Add some examples.
7. Improved letter indentation.

Here is the whole help message:
```
$ ./bin/kyuubi-beeline --help
Usage: kyuubi-beeline <options>. 

Options:
  -u <database url>               The JDBC URL to connect to.
  -c <named url>                  The named JDBC URL to connect to,
                                  which should be present in beeline-site.xml
                                  as the value of beeline.kyuubi.jdbc.url.<namedUrl>.

  -r                              Reconnect to last saved connect url (in conjunction with !save).
  -n <username>                   The username to connect as.
  -p <password>                   The password to connect as.
  -d <driver class>               The driver class to use.
  -i <init file>                  Script file for initialization.
  -e <query>                      Query that should be executed.
  -f <exec file>                  Script file that should be executed.
  -w, --password-file <file>      The password file to read password from.
  --hiveconf property=value       Use value for given property.
  --hivevar name=value            Hive variable name and value.
                                  This is Hive specific settings in which variables
                                  can be set at session level and referenced in Hive
                                  commands or queries.

  --property-file=<property file> The file to read connection properties (url, driver, user, password) from.
  --color=[true|false]            Control whether color is used for display.
  --showHeader=[true|false]       Show column names in query results.
  --escapeCRLF=[true|false]       Show carriage return and line feeds in query results as escaped \r and \n. 
  --headerInterval=ROWS;          The interval between which heades are displayed.
  --fastConnect=[true|false]      Skip building table/column list for tab-completion.
  --autoCommit=[true|false]       Enable/disable automatic transaction commit.
  --verbose=[true|false]          Show verbose error messages and debug info.
  --showWarnings=[true|false]     Display connection warnings.
  --showDbInPrompt=[true|false]   Display the current database name in the prompt.
  --showNestedErrs=[true|false]   Display nested errors.
  --numberFormat=[pattern]        Format numbers using DecimalFormat pattern.
  --force=[true|false]            Continue running script even after errors.
  --maxWidth=MAXWIDTH             The maximum width of the terminal.
  --maxColumnWidth=MAXCOLWIDTH    The maximum width to use when displaying columns.
  --silent=[true|false]           Be more silent.
  --autosave=[true|false]         Automatically save preferences.
  --outputformat=<format mode>    Format mode for result display.
                                  The available options ars [table|vertical|csv2|tsv2|dsv|csv|tsv|json|jsonfile]. 
                                  Note that csv, and tsv are deprecated, use csv2, tsv2 instead.

  --incremental=[true|false]      Defaults to false. When set to false, the entire result set
                                  is fetched and buffered before being displayed, yielding optimal
                                  display column sizing. When set to true, result rows are displayed
                                  immediately as they are fetched, yielding lower latency and
                                  memory usage at the price of extra display column padding.
                                  Setting --incremental=true is recommended if you encounter an OutOfMemory
                                  on the client side (due to the fetched result set size being large).
                                  Only applicable if --outputformat=table.

  --incrementalBufferRows=NUMROWS The number of rows to buffer when printing rows on stdout,
                                  defaults to 1000; only applicable if --incremental=true
                                  and --outputformat=table.

  --truncateTable=[true|false]    Truncate table column when it exceeds length.
  --delimiterForDSV=DELIMITER     Specify the delimiter for delimiter-separated values output format (default: |).
  --isolation=LEVEL               Set the transaction isolation level.
  --nullemptystring=[true|false]  Set to true to get historic behavior of printing null as empty string.
  --maxHistoryRows=MAXHISTORYROWS The maximum number of rows to store beeline history.
  --delimiter=DELIMITER           Set the query delimiter; multi-char delimiters are allowed, but quotation
                                  marks, slashes, and -- are not allowed (default: ;).

  --convertBinaryArrayToString=[true|false]
                                  Display binary column data as string or as byte array.

  --python-mode                   Execute python code/script.
  -h, --help                      Display this message.

Examples:
  1. Connect using simple authentication to Kyuubi Server on localhost:10009.
  $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 -n username

  2. Connect using simple authentication to Kyuubi Server on kyuubi.local:10009 using -n for username and -p for password.
  $ kyuubi-beeline -n username -p password -u jdbc:kyuubi://kyuubi.local:10009

  3. Connect using Kerberos authentication with kyuubi/localhost@mydomain.com as Kyuubi Server principal(kinit is required before connection).
  $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"

  4. Connect using Kerberos authentication using principal and keytab directly.
  $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiClientPrincipal=user@mydomain.com;kyuubiClientKeytab=/local/path/client.keytab;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"

  5. Connect using SSL connection to Kyuubi Server on localhost:10009.
  $ kyuubi-beeline -u "jdbc:kyuubi://localhost:10009/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"

  6. Connect using LDAP authentication.
  $ kyuubi-beeline -u jdbc:kyuubi://kyuubi.local:10009/default -n ldap-username -p ldap-password

  7. Connect using the ZooKeeper address to Kyuubi HA cluster.
  $ kyuubi-beeline -u "jdbc:kyuubi://zk1:2181,zk2:2181,zk3:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=kyuubi" -n username
```
If you have any suggested revisions, I will promptly respond and make the necessary adjustments.

## Types of changes :bookmark:

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Related Unit Tests

---

# Checklist 📝

- [ ] This patch was not authored or co-authored using [Generative Tooling](https://www.apache.org/legal/generative-tooling.html)

**Be nice. Be informative.**

### CODE REVIEW

File: kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java [hunk 1]
```diff
@@ -80,8 +80,6 @@ public KyuubiBeeLine(boolean isBeeLine) {
   @Override
   void usage() {
     super.usage();
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java:82]: we can eliminate this override then

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties [hunk 2]
```diff
@@ -154,73 +154,76 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
...
-\  --isolation=LEVEL               set the transaction isolation level\n \
-\  --nullemptystring=[true/false]  set to true to get historic behavior of printing null as empty string\n \
-\  --maxHistoryRows=MAXHISTORYROWS The maximum number of rows to store beeline history.\n \
-\  --delimiter=DELIMITER           set the query delimiter; multi-char delimiters are allowed, but quotation\n \
-\                                  marks, slashes, and -- are not allowed; defaults to ;\n \
-\  --convertBinaryArrayToString=[true/false]    display binary column data as string or as byte array \n \
-\  --help                          display this message\n \
+cmd-usage: Usage: kyuubi-beeline <options>. \n\n\
+Options:\n \
+\  -u <database url>                 The JDBC URL to connect to.\n \
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/resources/BeeLine.properties:159]: could you please keep the spaces? I don't see much benefit with extra two spaces

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties [hunk 3]
```diff
@@ -154,73 +154,76 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
...
+\   1. Connect using simple authentication to KyuubiServer on localhost:10009\n \
+\   $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 username password\n\n \
+\   2. Connect using simple authentication to KyuubiServer on kyuubi.local:10009 using -n for username and -p for password\n \
+\   $ kyuubi-beeline -n username -p password -u jdbc:kyuubi://kyuubi.local:10009\n\n \
+\   3. Connect using Kerberos authentication with hive/localhost@mydomain.com as KyuubiServer principal\n \
+\   $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;principal=hive/localhost@mydomain.com"\n\n \
+\   4. Connect using SSL connection to KyuubiServer on localhost at 10009\n \
+\   $ kyuubi-beeline "jdbc:kyuubi://localhost:10009/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n \
 \   5. Connect using LDAP authentication\n \
-\   $ beeline -u jdbc:hive2://hs2.local:10013/default <ldap-username> <ldap-password>\n \
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/resources/BeeLine.properties:226]: I remember this example does not work (I haven't verified it by myself)
https://issues.apache.org/jira/browse/HIVE-21239

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties [hunk 4]
```diff
@@ -154,73 +154,76 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
...
-\   3. Connect using Kerberos authentication with hive/localhost@mydomain.com as HiveServer2 principal\n \
-\   $ beeline -u "jdbc:hive2://hs2.local:10013/default;principal=hive/localhost@mydomain.com"\n\n \
-\   4. Connect using SSL connection to HiveServer2 on localhost at 10000\n \
-\   $ beeline "jdbc:hive2://localhost:10000/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n \
+\   1. Connect using simple authentication to KyuubiServer on localhost:10009\n \
+\   $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 username password\n\n \
+\   2. Connect using simple authentication to KyuubiServer on kyuubi.local:10009 using -n for username and -p for password\n \
+\   $ kyuubi-beeline -n username -p password -u jdbc:kyuubi://kyuubi.local:10009\n\n \
+\   3. Connect using Kerberos authentication with hive/localhost@mydomain.com as KyuubiServer principal\n \
+\   $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;principal=hive/localhost@mydomain.com"\n\n \
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/resources/BeeLine.properties:225]: actually, the `principal` is a confusing name, I see lots of users new to kerberos think it is the client principal name. we made some improvements on Kyuubi JDBC driver on this area, see https://kyuubi.readthedocs.io/en/master/client/jdbc/kyuubi_jdbc.html
```suggestion
\   3. Connect using Kerberos authentication with `kyuubi/localhost@mydomain.com` as KyuubiServer principal (kinit is required before connection)\n \
\   $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"\n\n \
```

And Kyuubi also supports using keytab for Kerberos authentication directly (without required to perform kinit to generate TGT cache)

```
kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiClientPrincipal=user@mydomain.com;kyuubiClientKeytab=/local/path/client.keytab;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"
```

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties [hunk 5]
```diff
@@ -154,73 +154,76 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
...
+\   4. Connect using SSL connection to KyuubiServer on localhost at 10009\n \
+\   $ kyuubi-beeline "jdbc:kyuubi://localhost:10009/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n \
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/resources/BeeLine.properties:227]: can we add an example for zookeeper HA cases?

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties (hunk 2 above)
Comment (dupen01) [reply kyuubi-hive-beeline/src/main/resources/BeeLine.properties:159]: Yeah. I will remove these two extra spaces in the next commit.

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties (hunk 3 above)
Comment (dupen01) [reply kyuubi-hive-beeline/src/main/resources/BeeLine.properties:226]: Yes, I just checked this JIRA as well, and according to the reply within it, when using LDAP authentication for the connection, `-n <ldap-username> -p <ldap-password>` is required for proper functioning. Furthermore, I examined the master branch code of Hive: `BeeLine.properties`, and indeed, it has already been modified to `$ beeline -n ldap-username -p ldap-password -u "jdbc:hive2://localhost:10000/default"`.

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties (hunk 5 above)
Comment (dupen01) [reply kyuubi-hive-beeline/src/main/resources/BeeLine.properties:227]: Of course.

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties [hunk 6]
```diff
@@ -154,73 +154,80 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
...
+\  3. Connect using Kerberos authentication with kyuubi/localhost@mydomain.com as Kyuubi Server principal(kinit is required before connection).\n\
+\  $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"\n\n\
+\  4. Connect using Kerberos authentication using principal and keytab directly.\n\
+\  $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiClientPrincipal=user@mydomain.com;kyuubiClientKeytab=/local/path/client.keytab;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"\n\n\
+\  5. Connect using SSL connection to Kyuubi Server on localhost:10009.\n\
+\  $ kyuubi-beeline -u "jdbc:kyuubi://localhost:10009/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n\
+\  6. Connect using LDAP authentication.\n\
+\  $ kyuubi-beeline -u jdbc:kyuubi://kyuubi.local:10009/default -n ldap-username -p ldap-password\n\n\
+\  7. Connect using the ZooKeeper address to Kyuubi HA Server on localhost:2181.\n\
+\  $ kyuubi-beeline -u "jdbc:kyuubi://localhost:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=kyuubi" -n username
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/resources/BeeLine.properties:233]:
```suggestion
\  $ kyuubi-beeline -u "jdbc:kyuubi://zk1:2181,zk2:2181,zk3:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=kyuubi" -n username
```

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties [hunk 7]
```diff
@@ -154,73 +154,80 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
...
+\  $ kyuubi-beeline -n username -p password -u jdbc:kyuubi://kyuubi.local:10009\n\n\
+\  3. Connect using Kerberos authentication with kyuubi/localhost@mydomain.com as Kyuubi Server principal(kinit is required before connection).\n\
+\  $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"\n\n\
+\  4. Connect using Kerberos authentication using principal and keytab directly.\n\
+\  $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiClientPrincipal=user@mydomain.com;kyuubiClientKeytab=/local/path/client.keytab;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"\n\n\
+\  5. Connect using SSL connection to Kyuubi Server on localhost:10009.\n\
+\  $ kyuubi-beeline -u "jdbc:kyuubi://localhost:10009/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n\
+\  6. Connect using LDAP authentication.\n\
+\  $ kyuubi-beeline -u jdbc:kyuubi://kyuubi.local:10009/default -n ldap-username -p ldap-password\n\n\
+\  7. Connect using the ZooKeeper address to Kyuubi HA Server on localhost:2181.\n\
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/resources/BeeLine.properties:232]:
```suggestion
\  7. Connect using the ZooKeeper address to Kyuubi HA cluster.\n\
```

File: kyuubi-hive-beeline/src/main/resources/BeeLine.properties [hunk 8]
```diff
@@ -154,73 +154,80 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
...
+\  --delimiter=DELIMITER           Set the query delimiter; multi-char delimiters are allowed, but quotation\n\
+\                                  marks, slashes, and -- are not allowed (default: ;).\n\n\
+\  --convertBinaryArrayToString=[true|false]\n\
+\                                  Display binary column data as string or as byte array.\n\n\
+\  --python-mode                   Execute python code/script.\n\
+\  -h, --help                      Display this message.\n\
+\n\
+Examples:\n\
+\  1. Connect using simple authentication to Kyuubi Server on localhost:10009.\n\
+\  $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 <username> <password>\n\n\
```
Comment (pan3793) [inline kyuubi-hive-beeline/src/main/resources/BeeLine.properties:221]: same as LDAP, it does not work too. let's replace it with the following example
```suggestion
\  $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 -n username\n\n\
```

File: kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java (hunk 1 above)
Comment (pan3793) [reply kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java:82]: I mean delete the method `usage`, we don't need to override a method that just calls `super.<method-name>`

Comment (dupen01) [reply kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java]: Ok, got it. I've made this modification and committed it.

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merged to master/1.9

**Pull request closed without merging**